apps/dashboard/src/context/opt-in-provider.tsx

```diff
@@ -11,6 +11,10 @@ export const OptInProvider = (props: PropsWithChildren) => {
     localStorage.setItem('mantine-theme', 'light');
   }, []);
 
+  if (isLoaded && status === null) {
+    return null;
+  }
```

Comment:
Will this flow ensure that later in the flow the invited user will be marked as opt_in? I just don't remember if we mark it opt in only for when creating the org, and not during the invite process